packages/x-components/tests/e2e/cucumber/tagging/tagging.spec.ts

```diff
@@ -1,22 +1,28 @@
 import { Given, Then, When } from 'cypress-cucumber-preprocessor/steps';
 
-Given('a URL with query parameter {string}', (query: string) => {
-  cy.visit(`/?useMockedAdapter=true&q=${query}`);
+Given('a tracking API with a known response', () => {
```

Comment:
Would it make sense to move this to `mocked-responses.spec.ts` with the other mocked tracking